{"name": "Попытка принять заказ с указанием неверного id курьера", "status": "passed", "start": 1706508488086, "stop": 1706508488719, "uuid": "dda7e91e-62b8-442a-88c0-69ce6ea4b987", "historyId": "e0236b6b2ccea89a76c9220d1b02f651", "testCaseId": "e0236b6b2ccea89a76c9220d1b02f651", "fullName": "tests.test_take_order.TestTakeOrder#test_take_order_with_incorrect_id_courier", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_take_order"}, {"name": "subSuite", "value": "TestTakeOrder"}, {"name": "host", "value": "MBP-Ilya.cn.ru_novotelecom.ru"}, {"name": "thread", "value": "11014-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_take_order"}]}